YellowNotes.Core/Email/EmailConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace YellowNotes.Core.Email
+{
+    public class EmailConfiguration : IEmailConfiguration
```

Comment:
There is no need to create interface for dto, models, configurations etc, because there is no business logic.